src/types/openspace-api-js.d.ts

```diff
@@ -449,7 +449,7 @@ export interface openspace {
   /**
    * Returns the value the property, identified by the provided URI. Deprecated in favor of the 'propertyValue' function
    */
-  propertyValue: () => Promise<void>
+  propertyValue: (uri: string) => Promise<unknown>
```

Comment:
Use the new type :) 
```suggestion
  propertyValue: (uri: Uri) => Promise<unknown>
```

Reply:
I want to move away from manually editing this file as much as possible since any change we make will be overwritten when we generate a new file. So I'll leave it out for now

Reply:
Ah, sorry!  Didn't see that it was that file :) 

Ignore this comment then